test/dateTime.js

```diff
@@ -305,26 +293,26 @@ describe('dateTime', () => {
 			{locale: 'en-CA', expect: ['01:28', '1:28 AM', '01:28 EST', '1:28 AM EST', '13:52', '1:52 PM', '13:52 EST', '1:52 PM EST']},
 			{locale: 'en-GB', expect: ['01:28', '1:28 AM', '01:28 EST', '1:28 AM EST', '13:52', '1:52 PM', '13:52 EST', '1:52 PM EST']},
 			{locale: 'en-US', expect: ['01:28', '1:28 AM', '01:28 EST', '1:28 AM EST', '13:52', '1:52 PM', '13:52 EST', '1:52 PM EST']},
-			{locale: 'es', expect: ['1:28'/*01:28*/, '1:28'/*1:28 a. m.*/, '1:28 EST'/*01:28 EST*/, '1:28 EST'/*1:28 a. m. EST*/, '13:52', '13:52'/*1:52 p. m.*/, '13:52 EST', '13:52 EST'/*1:52 p. m. EST*/]},
+			{locale: 'es', expect: ['01:28', '1:28 a. m.', '01:28 EST', '1:28 a. m. EST', '13:52', '1:52 p. m.', '13:52 EST', '1:52 p. m. EST']},
 			{locale: 'es-MX', expect: ['01:28', '1:28 a. m.', '01:28 EST', '1:28 a. m. EST', '13:52', '1:52 p. m.', '13:52 EST', '1:52 p. m. EST']},
-			{locale: 'fr', expect: ['01 h 28', '01 h 28', '01 h 28 EST', '01 h 28 EST', '13 h 52', '13 h 52', '13 h 52 EST', '13 h 52 EST']},
-			{locale: 'fr-FR', expect: ['01 h 28', '01 h 28', '01 h 28 EST', '01 h 28 EST', '13 h 52', '13 h 52', '13 h 52 EST', '13 h 52 EST']},
-			{locale: 'fr-CA', expect: ['01 h 28', '01 h 28', '01 h 28 EST', '01 h 28 EST', '13 h 52', '13 h 52', '13 h 52 EST', '13 h 52 EST']},
-			{locale: 'ja', expect: ['1:28', '1:28'/*1:28 午前*/, '1:28 EST', '1:28 EST'/*1:28 午前 EST*/, '13:52', '13:52'/*1:52 午後*/, '13:52 EST', '13:52 EST'/*1:52 午後 EST*/]},
-			{locale: 'ja-JP', expect: ['1:28', '1:28'/*1:28 午前*/, '1:28 EST', '1:28 EST'/*1:28 午前 EST*/, '13:52', '13:52'/*1:52 午後*/, '13:52 EST', '13:52 EST'/*1:52 午後 EST*/]},
-			{locale: 'ko', expect: ['오전 1:28'/*01:28*/, '오전 1:28', '오전 1:28 EST'/*01:28 EST*/, '오전 1:28 EST', '오후 1:52'/*13:52*/, '오후 1:52', '오후 1:52 EST'/*13:52 EST*/, '오후 1:52 EST']},
-			{locale: 'ko-KR', expect: ['오전 1:28'/*01:28*/, '오전 1:28', '오전 1:28 EST'/*01:28 EST*/, '오전 1:28 EST', '오후 1:52'/*13:52*/, '오후 1:52', '오후 1:52 EST'/*13:52 EST*/, '오후 1:52 EST']},
```

Comment:
This now all correctly match the commented-out values, which were what we were expecting.